### ISSUE TITLE
gnome-control-center installed brakes wayland programs

### ISSUE THREAD

**Issue by Pablo1107** [NixOS/nixpkgs#119769]
**Describe the bug**
If gnome-control-center is installed on my Arch system, all wayland programs installed via Nix fails to start giving the following error:

```
GLib-GIO-ERROR **: 00:02:57.432: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'
zsh: trace trap (core dumped)
```

**To Reproduce**
1. Install gnome-control-center on Arch
2. Run some wayland program from nixpkgs. Example: `nix run nixpkgs.wofi -c wofi --show run`
3. See above error.
4. Uninstall gnome-control-center and repeat step 2.
5. See no errors.

**Expected behavior**
I don't know what to expect honestly, as I keep pacman installed packages to a minimum, but didn't knew that having that package installed made that error happens and for what reason it does.

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.11.13-arch1-1, Arch Linux, noversion`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.10`
 - channels(pablo): `"home-manager, nixpkgs-21.05pre282987.294d1925af6"`
 - channels(root): `"nixpkgs-21.05pre282987.294d1925af6, nixos-20.09.3827.389249fa9b3"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

**Comment by sceglimilano** [comment]
Same issue here:
I can't run any nix installed program.
I use GNOME 3 with Wayland on archlinux.
VGA: AMD RX 5700XT

`these paths will be fetched (0.51 MiB download, 2.48 MiB unpacked):
  /nix/store/059m11gqd5yjjj9wpmlhqn6i9rl0zya2-bash-interactive-4.4-p23-man
  /nix/store/0vxmxszx1v9j4ffpvc1m21p0zwhgrjz2-bash-interactive-4.4-p23-doc
  /nix/store/1hg7ssk7dzpmccsz018p8vb9jr3pq47i-nix-info
  /nix/store/j7g6w8n9yvlraz45cd8myf60yrbqdlkr-bash-interactive-4.4-p23-info
  /nix/store/y8fkm1p4s3al9p29kxargz30h1hkjlwj-bash-interactive-4.4-p23-dev
copying path '/nix/store/0vxmxszx1v9j4ffpvc1m21p0zwhgrjz2-bash-interactive-4.4-p23-doc' from 'https://cache.nixos.org'...
copying path '/nix/store/y8fkm1p4s3al9p29kxargz30h1hkjlwj-bash-interactive-4.4-p23-dev' from 'https://cache.nixos.org'...
copying path '/nix/store/j7g6w8n9yvlraz45cd8myf60yrbqdlkr-bash-interactive-4.4-p23-info' from 'https://cache.nixos.org'...
copying path '/nix/store/059m11gqd5yjjj9wpmlhqn6i9rl0zya2-bash-interactive-4.4-p23-man' from 'https://cache.nixos.org'...
copying path '/nix/store/1hg7ssk7dzpmccsz018p8vb9jr3pq47i-nix-info' from 'https://cache.nixos.org'...
bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
/nix/store/6737cq9nvp4k5r70qcgf61004r0l2g3v-bash-4.4-p23/bin/bash: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
 - system: `"x86_64-linux"`
 - host os: `Linux 5.11.16-arch1-1, Arch Linux, noversion`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.6`
 - channels(root): `"nixpkgs-20.09pre238494.96745f02283"`
 - channels(archio): `"nixpkgs-21.05pre283554.d9448c95c5d"`
 - channels(lvg0): `"nixpkgs-21.03pre256488.b3abbd61f52"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
`

**Comment by Pablo1107** [comment]
> Same issue here:
> I can't run any nix installed program.
> I use GNOME 3 with Wayland on archlinux.
> VGA: AMD RX 5700XT
> 
> ```
>  - system: `"x86_64-linux"`
>  - host os: `Linux 5.11.16-arch1-1, Arch Linux, noversion`
>  - multi-user?: `yes`
>  - sandbox: `yes`
>  - version: `nix-env (Nix) 2.3.6`
>  - channels(root): `"nixpkgs-20.09pre238494.96745f02283"`
>  - channels(archio): `"nixpkgs-21.05pre283554.d9448c95c5d"`
>  - channels(lvg0): `"nixpkgs-21.03pre256488.b3abbd61f52"`
>  - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
> ```

Did you installed GNOME through Nix or Pacman?

**Comment by jtojnar** [comment]
GNOME 40 has been since merged so this should be fixed on nixos-unstable.

**Comment by mogorman** [comment]
will these issues be fixed in nixos 21.05? I have similiar issue when trying to run tilix now

**Comment by jtojnar** [comment]
Yes, the GTK version is in 21.05.

**Comment by mogorman** [comment]
thanks. i had an old package installed somehow, updating solved my problem on 21.05. thanks
